### ISSUE TITLE
[Enhancement] Label: Underline + Strike

### ISSUE THREAD

**Issue by kingces95** [xamarin/Xamarin.Forms#1674]
## Rationale

Add enum to enable strike-through and underline to labels (and buttons). 

## Implementation

```csharp
[Flags]
public enum FontAttributes
{
	None = 0,
	Bold = 1 << 0,
	Italic = 1 << 1,
	Underline = 1 << 2,
	Strike = 1 << 3
}
```

## Expected Result

Label and Button text respects new flags added to FontAttribute. 

## Implications for CSS

None.

## Backward Compatibility

None.

## Difficulty : Easy

**Comment by adamped** [comment]
Label Underline is referenced here: https://github.com/xamarin/Xamarin.Forms/issues/1632

**Comment by hartez** [comment]
@jason @kingces95 We need to decide whether this issue and https://github.com/xamarin/Xamarin.Forms/issues/1632 should be one issue, or whether this issue should drop 'underline' and just focus on 'strike-through'.

**Comment by hartez** [comment]
@kingces95 Under CSS implications, this should probably require adding support for `text-decoration` `line-through`.

**Comment by samhouts** [comment]
Since we're updating the same enum and same code in both feature requests, I vote we combine the issues to prevent conflicts. We can close #1632 and redirect here.

**Comment by hartez** [comment]
Okay, merged this spec into #1632.
